web/views/layout_view.ex

```diff
@@ -2,6 +2,7 @@ defmodule Appointments.LayoutView do
   use Appointments.Web, :view
 
   def active?(conn, path) do
-    if conn.request_path == "/#{path}", do: "active", else: ''
+    [_ | [resource | _]] = String.split(conn.request_path, "/")
```

Comment:
In that case we would have something like this:
`[ a | [ b | c ]] = String.split("/", "/")` such that  `a=""`, `b=""` and `c=[]`.
But I guess would could make this prettier ;-)